src/wallet/wallet.cpp

```diff
@@ -1932,33 +1932,26 @@ std::set<uint256> CWalletTx::GetConflicts() const
     return result;
 }
 
+CAmount CWalletTx::GetCachable(AmountType type, const isminefilter& filter, bool recalculate) const
```

Comment:
Note: Could call this `GetAmount` instead of `GetCachable` as suggested previously https://github.com/bitcoin/bitcoin/pull/15780#discussion_r275597994 because this returns an amount and now handles caching internally instead of taking a cache reference.

Reply:
The `recalculate` bool is for the cache, though. Maybe `GetCachableAmount` is better. Using that.